multimodal/src/autogluon/multimodal/constants.py

```diff
@@ -64,8 +64,21 @@
 SCORE = "score"
 LOGIT_SCALE = "logit_scale"
 
-# Metric
+# Metric for Object Detection
 MAP = "map"
```

Comment:
One question, MAP standards for mean average precision, will it be confusing to other people when they see MEAN_AVERAGE_PRECISION in line 82?

Reply:
Good point, edited the code to make MEAN_AVERAGE_PRECISION also work (=MAP).